doc/getting-started.md

```diff
@@ -8,31 +8,46 @@ This short introduction will teach you the basics of using your Almond.
 
 ## What is Almond?
 
-Almond is the magic virtual assistant that you access through your phone. She can
-help you configure your things, execute actions on them, install apps based on
-your description of the behavior you want.
+Almond is the open virtual assistant that you can access through both of 
+your phone and the [web](/me). It can help you configure your devices, accounts, 
```

Comment:
The user is unlikely to have an account at this point. How about:
```suggestion
your phone and the [web](/about/get-almond). It can help you configure your devices, accounts, 
```